crates/sui-core/src/safe_client.rs

```diff
@@ -56,7 +68,8 @@ impl<C> SafeClient<C> {
     ) -> SuiResult {
         if let Some(signed_transaction) = &response.signed_transaction {
             // Check the transaction signature
-            signed_transaction.verify(&self.committee)?;
+            signed_transaction
+                .verify(&self.get_committee(&signed_transaction.auth_sign_info.epoch)?)?;
```

Comment:
I think the committee should be the same throughout the call, so why not take it once, and re-use it to make this clear? (And to avoid it changing half way through).